src/zope/app/tree/utils.py

```diff
@@ -14,13 +14,15 @@
 """Static tree utilities
 
 """
+from binascii import b2a_base64, a2b_base64
 __docformat__ = 'restructuredtext'
```

Comment:
no empty line between imports and first line of code? Is this our standard?

Reply:
We have no enforced standard outside `zope-product` (see above.) But this looks weird. Fixed in https://github.com/zopefoundation/zope.app.tree/pull/9/commits/90f2305ed68c38fdc6672d069c40de8befd1227f.